content/cloud-docs/architectural-details/data-security.mdx

```diff
@@ -20,6 +20,7 @@ seriously. This table lists which parts of the Terraform Cloud and Terraform Ent
 | Terraform Plan Result                | Blob Storage | Vault Transit Encryption |
 | Terraform State                      | Blob Storage | Vault Transit Encryption |
 | Terraform Logs                       | Blob Storage | Vault Transit Encryption |
+| Private Module Registry Modules      | Blob Storage | Vault Transit Encryption |
```

Comment:
@chrisfishwood how are child mods stored - are they references to the git repos with metadata object stored in object store?